packages/react-scripts/template/src/components/example/FrontierDocsCard.js

```diff
@@ -32,4 +33,13 @@ const FrontierDocsCard = ({ logoColor, animationDuration, handleLogoClick }) =>
   </Card>
 )
 
+FrontierDocsCard.propTypes = {
+  /** The logoColor is used to set the color of the Logo component. */
+  logoColor: PropTypes.string.isRequired,
```

Comment:
this isn't required. it defaults to black I think.
also, the animationDuration also have a default.

Reply:
agreed, not required